packages/demo/src/app/home/home.component.html

```diff
@@ -843,9 +858,9 @@ <h5 class="mt-4 h6 bold">Background</h5>
                       />
                       <!-- prettier-ignore -->
                       <label class="form-check-label" for="bootstrapbackgrounds_textcolor">
-                      <span class="change-badge bg-danger">breaking</span> Background classes such as <code>.bg-[colorname]</code>, now also automatically define the inner <span class="highlight">text-color</span> with <code>var(--post-contrast-color)</code>.<br>
-                      Therefore an explicit <span class="highlight">text-color</span> styling is not required anymore.
-                      </label>
+                        <span class="change-badge bg-danger">breaking</span> Background classes such as <code>.bg-[colorname]</code>, now also automatically define the inner <span class="highlight">text-color</span> with <code>var(--post-contrast-color)</code>.<br>
```

Comment:
That entry is suspicious. A `text-color` class never existed. `text-auto` did and has an automatic migration available which is already documented.